platforms/r3-corda-ent/charts/corda-ent-node/values.yaml

```diff
@@ -173,7 +173,7 @@ networkServices:
 # external firewall (Bridge and Float)
 firewall:
   # enabled is a boolean value for the above function
-  enabled:
+  enabled: false

```

Comment:
Helm linting: cannot set a boolean variable to empty for independent Helm chart deployment.